harness/determined/pytorch/_pytorch_context.py

```diff
@@ -133,6 +134,8 @@ def __init__(
 
         self._tbd_writer = None  # type: Optional[Any]
         self._enable_tensorboard_logging = enable_tensorboard_logging
+        # Timestamp for batching TensorBoard uploads
+        self._last_tb_reset_ts: Optional[float] = None
```

Comment:
I recommend avoiding `Optional` and defaulting to `0.0` instead; it would simplify the code.  As long as nobody runs this code before 1970, I suppose.

Reply:
this would cause the first event file to be overwritten. the initial reset ts needs to be a value that _doesn't_ cause the reset logic to trigger.

otherwise what happens is we would reset the initial file then open a new one (usually within the same second), and the original file would get overwritten.